### PR TITLE
Auto tag dumpers with `bazinga_geocoder.dumper`

### PR DESCRIPTION
**Pull request by norkunas**

### CODE REVIEW

File: Resources/config/services.yml [hunk 1]
```diff
@@ -1,27 +1,15 @@
...
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Wkb:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
-
-    Geocoder\Dumper\Wkt:
-        public: true
-        tags: ['bazinga_geocoder.dumper']
+    _instanceof:
```
Comment (Nyholm) [inline Resources/config/services.yml]: Is this needed?

File: Resources/config/services.yml [hunk 2]
```diff
@@ -1,27 +1,15 @@
...
+        Geocoder\Dumper\Dumper:
+            tags: ['bazinga_geocoder.dumper']
+            public: true
+
+    Geocoder\Dumper\GeoArray: ~
+    Geocoder\Dumper\GeoJson: ~
```
Comment (Nyholm) [inline Resources/config/services.yml]: Are these needed?

File: Resources/config/services.yml (hunk 1 above)
Comment (norkunas) [reply Resources/config/services.yml]: Yes, just simplifying configuration instead of repeating

File: Resources/config/services.yml (hunk 2 above)
Comment (norkunas) [reply Resources/config/services.yml]: Dumpers would not be registered otherwise :thinking:

Comment (Nyholm) [reply Resources/config/services.yml]: Oh, sorry. It is too early in the morning. My two comments are invalid.

**Pull request merged**